### ISSUE TITLE
[BioDrop Images] <change LinkFree images in QuickStart JSON guide>

### ISSUE THREAD

**Issue by goobric** [EddieHubCommunity/BioDrop#9697]
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

on the QuickStart JSON url 'https://www.biodrop.io/docs/quickstart-json' all of the images to each of the steps to setup up a BioDrop profile via a JSON file, all reference LinkFree, and not BioDrop

For a new person to tech, with no coding skills, who wants to try the JSON route for BioDrop, might find the 'Quick Start json' instructions could be confusing, because the following steps to create this file refers to 'LinkFree' in the images following the numbers instructions below: 

step 02: visit BioDrop, which implies this will redirect to 'LinkFree'
step 03: refers to 'LinkFree' repo
step 04: says 'fork LinkFree' new repo will be 'LinkFree'
step 05: says 'fork BioDrop' refers to 'LinkFree' as new fork
step 07: refers to 'LinkFree' twice
step 13: refers to 'LinkFree' twice
step 15: refers to 'LinkFree' twice
step 17: refers to 'LinkFree'
step 19: refers to 'LinkFree'

### Proposed Solution

Proposal:
Update the images below each of the instructions to reflect 'BioDrop' instead of 'LinkFree'
This then negates any ambiguity as to which project the person is using, to publicise and advertise themselves.

### Screenshots

![BD_LF02](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/def7357c-b269-41c8-bf9a-6466183f3155)
![BD_LF03](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/4620ba45-1521-429f-95ea-37d1c4969459)
![BD_LF04](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/7ade3b64-85a1-4b14-a226-d628d842f5d1)
![BD_LF05](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/df07f469-61da-4004-990c-fe57291af21a)
![BD_LF07](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/40ff7723-6367-4e21-982d-3faa0b0f9758)
![BD_LF13](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/91f4f587-cc4a-4c21-89f0-7e7c41e2c099)
![BD_LF15](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/62e0eb9b-c4da-41fe-8682-ffa10a07b685)
![BD_LF17](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/34048a99-091d-404f-9b56-bc3f7da50b27)
![BD_LF19](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/c5a54405-42ba-4968-a272-fca78e44aff6)

### Do you want to work on this issue?

Yes

### If you want to work on this issue...

I would use updated images referring to BioDrop from the current BioDrop repo in each step, for setting up a BioDrop profile via a JSON file.

**Comment by SaraJaoude** [comment]
Thanks for raising @goobric  - I will assign this to you. 

As I have deleted my json file (as I am using forms), for the example screenshots please could you use Eddie Jaoude's profile. 

I am also making a small edit to the title to change "references" to "images".

**Comment by goobric** [comment]
hi @SaraJaoude 
thank you so much for assigning this issue 9697 to me.
May I ask; the LinkFree images that I need to replace; are there any specific images that you would like me to use?
Or, during the process of creating my BioDrop profile via the JSON route, I will take screen images during each step; and then either send them to you, or store them in a repo for you to access, to inform me if they are what the project needs.

**Comment by eddiejaoude** [comment]
Great questions Mikal 👍 

> the LinkFree images that I need to replace; are there any specific images that you would like me to use?
> Or, during the process of creating my BioDrop profile via the JSON route, I will take screen images during each step

I can share the screenshots here, so they are consistent throughout the docs and also the doc videos

> send them to you, or store them in a repo for you to access, to inform me if they are what the project needs

I will drop the screenshots here in another comment, and GitHub will generate a url you can use to update the current screenshots

I hope that makes sense

**Comment by goobric** [comment]
thank you so much @eddiejaoude much appreciated; being consistent throughout the documentation helps the user to re-reference/re-read the steps and aids the understanding of generating the required json file.

**Comment by goobric** [comment]
thanks @eddiejaoude when I am back from college, I will download the images; fork, clone, create a new branch, and start to work on the issue.
I searched the repo with 'quickstart-json' ; can I please confirm that this is the correct file and path:
BioDrop/pages/docs/quickstart-json.mdx
If so, I will change the URL references for the images on the current page, to the URL image references that you sent me?
![231031_BD_qsJSON](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/25782b91-1aa0-4f8d-82d9-0311c1329f9d)

**Comment by eddiejaoude** [comment]
That looks correct 👍  sure when you get time

**Comment by Pablohn26** [comment]
![2023-10-31_14-54](https://github.com/EddieHubCommunity/BioDrop/assets/1228418/0150038f-4702-4a24-9a92-b51aeaa7916e)
![2023-10-31_14-59](https://github.com/EddieHubCommunity/BioDrop/assets/1228418/f99a72d5-c142-4c2a-b388-12251d98b29e)
![2023-10-31_15-58](https://github.com/EddieHubCommunity/BioDrop/assets/1228418/52d141c7-24e8-40f5-9297-09412e8279d0)
![2023-10-31_16-00](https://github.com/EddieHubCommunity/BioDrop/assets/1228418/5dbd6335-ff77-4209-960b-7d0bc8a53f91)
![2023-10-31_16-03](https://github.com/EddieHubCommunity/BioDrop/assets/1228418/df59d7c4-1e3b-4c40-8d7b-aed24d56987f)
![Peek 31-10-2023 15-00](https://github.com/EddieHubCommunity/BioDrop/assets/1228418/551b3c6f-b33a-46e7-904d-3113ffca2c6c)


I left here the pictures that I took (I was going to submit a PR with the changes before I saw this issue), maybe they are useful for you

**Comment by goobric** [comment]
hi @eddiejaoude I have just tried to fork BioDrop; GitHub wants to create 'LinkFree' as the fork in my GH account; and not 'BioDrop' ! is this correct?
I have renamed the original LinkFree fork to LinkFree2; closed and re-opened GH and signed in.
I tried to fork BD, but keep receiving this message from GH: 'No available destinations to fork this repository.' ! see image
I am confused :-/  Are you able to explain why this is happens?
I can create a fork of other repositories, see image

I can create an amended 'quickstart-json.mdx' file, without forking, cloning, new branch, and PR; if this is easier.

**Comment by goobric** [comment]
![LF_BD_forkLinkFree](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/24ec157a-7a19-41a3-8811-e6e7b348a78d)
![LF_BD_forkLinkFree1](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/52a22039-2c2b-49e1-bc89-72f86512e84a)
![LF_BD_forkLinkFree3](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/0c1d0ee1-e20b-4dfa-a0a0-156dd36bf8c4)

**Comment by goobric** [comment]
@eddiejaoude sorry for asking so many questions, and querying so many things!  :-(

**Comment by goobric** [comment]
hi @Pablohn26 thanks for the images; Eddie has provided the images that will to be used within the amended json document.

**Comment by eddiejaoude** [comment]
@Pablohn26 I am not sure what those images are for? please read the discussion

**Comment by eddiejaoude** [comment]
no problem @goobric . Because of the rename I think GitHub has got itself a bit confused. If you have no changes you want to keep in your fork `LinkFree` (or the rename), please delete it under settings and then you will be able to fork `BioDrop` correctly

**Comment by goobric** [comment]
hi @SaraJaoude @eddiejaoude 
I deleted the LinkFree repo, and was able to fork the BioDrop repo; thank you so much.
Some additional information regarding the solution for raised issue:
In keeping with the initial images having a red border around specific words that the user needs to select; I have used this same
approach for the images where the user needs click, to highlight these words.

step 3 : I have drawn a red border around the word 'starred' on the new image for step3, and added a suffix _03.png

step 4 "Click on Fork" : the image given is after the word 'fork' has been clicked.
Proposal: use new image from step 3, but with a red border around the word 'fork'
I have duplicated the new image for step 3, and drawn a red border around the word 'fork' and added a suffix _04.png

step 5: Proposal: use two images for this step, and change the current sentence to:
Accept the default fork name, click on 'CREATE'. A forked copy of the BioDrop project is now open in your GitHub account
I have drawn a red border around the word 'CREATE' and added a suffix _05a.png

step 09 & 12 : no new accompanying images for these steps. Do you wish to use the current images for these steps?

step 15: I have drawn a red border around the word 'Create pull request' and added a suffix _15.png

step 17: no new accompanying image for this step. Do you wish to use the current image for this step?

I have attached the updated images, if these would be appropriate for the respective steps.
Please advise if you would like to implement the above proposals, or something else.

**Comment by goobric** [comment]
![279361179-e00b3df3-f55e-46ca-bb02-9e3603947f48_03](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/e8ca4e9c-978c-45cd-aa28-449d7dd89404)
![279361179-e00b3df3-f55e-46ca-bb02-9e3603947f48_04](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/fe6b4787-b768-49e3-a678-39643f9db25c)
![279361213-832445a5-b2ed-48d5-a633-82f6a6ec2b54_05a](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/e4a98d5e-8fc3-4dd9-87b3-5b7197aee342)
![279361309-354c0448-34e2-457e-b117-10c975ee5f69_15](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/752ab63c-ed16-4515-b48d-da3127d70eb2)

**Comment by goobric** [comment]
sorry, proposal for step 5, should be:
Accept the default fork name, click on 'Create fork'. A forked copy of the BioDrop project is now open in your GitHub account

**Comment by goobric** [comment]
Also, sorry, I should have mentioned a proposal change for the text in step 7
Current:
For example for username **"SaraJaoude"**:
> - _**Correct**_: `BioDrop/data/SaraJaoude.json` (including capitalisation in filename)
> - _**Incorrect**_: `BioDrop/data/sarajaoude.json` (filename capitalisation does not match username capitalisation)

Proposal:
For example for username **"eddieJaoude"**:

> - _**Correct**_: `BioDrop/data/EddieJaoude.json` (including capitalisation in filename)
> - _**Incorrect**_: `BioDrop/data/eddiejaoude.json` (filename capitalisation does not match username capitalisation)

**Comment by goobric** [comment]
I have corrected the text in step7 to reflect the image used in step7 (EddieHubCommunity/BioDrop/assets/624760/0f837e97-b079-4380-b451-8dfe23206842)

For example for username **"eddiejaoude"**:

> - _**Correct**_: `BioDrop/data/eddiejaoude.json` (this example: no capitalisation in filename)
> - _**Incorrect**_: `BioDrop/data/EddieJaoude.json` (filename capitalisation does not match username)

**Comment by SaraJaoude** [comment]
@goobric thanks for setting this all out.  To answer your questions:


> step 4 "Click on Fork" : the image given is after the word 'fork' has been clicked. Proposal: use new image from step 3, but with a red border around the word 'fork' I have duplicated the new image for step 3, and drawn a red border around the word 'fork' and added a suffix _04.png

Great idea


> step 5: Proposal: use two images for this step, and change the current sentence to: Accept the default fork name, click on 'CREATE'. A forked copy of the BioDrop project is now open in your GitHub account I have drawn a red border around the word 'CREATE' and added a suffix _05a.png

Sounds good

> step 09 & 12 : no new accompanying images for these steps. Do you wish to use the current images for these steps?

I think this also applies to Step 8 as well. Here is the new code Block for that step: 

```json
{
  "name": "Eddie Jaoude",
  "bio": "Open Source DevRel | Founder of **EddieHub** and [LinkFree](https://linkfree.io) | **GitHub** Star"
}
```

New code blocks and image for Step 9:

```json
{
  "name": "Eddie Jaoude",
  "bio": "Open Source DevRel | Founder of **EddieHub** and [LinkFree](https://linkfree.io) | **GitHub** Star",
  "links": [
    {
      "group": "Business",
      "name": "Business: DevRel Services",
      "url": "http://eddiejaoude.io",
      "icon": "FaLink"
    }
  ]
}
```

<img width="2228" alt="Screenshot 2023-11-01 at 13 24 43" src="https://github.com/EddieHubCommunity/BioDrop/assets/82668196/e47b89c9-f28e-4f06-a2d2-94a69bb386e2">

```json
"links": [
    {
      "group": "Business",
      "name": "Business: DevRel Services",
      "url": "http://eddiejaoude.io",
      "icon": "FaLink"
    },
    {
      "group": "Socials Primary",
      "name": "Twitter: Follow me",
      "url": "https://twitter.com/eddiejaoude",
      "icon": "FaTwitter",
      "color": "#00ACEE"
    },
]
```

New image for Step 12:

![Screenshot 2023-11-01 at 13 08 55](https://github.com/EddieHubCommunity/BioDrop/assets/82668196/ad2524c2-1fe3-469a-b5a2-cf436f1fe5f8)

> step 15: I have drawn a red border around the word 'Create pull request' and added a suffix _15.png

Good idea

> step 17: no new accompanying image for this step. Do you wish to use the current image for this step?

Step 17 new image:

**Comment by goobric** [comment]
thank you for the update and the additional resources to complete the solution for the issue.
when I am back from college, I will continue to update my clone and branch with the solution.

**Comment by goobric** [comment]
I have implemented all of the changes.
regarding Step 19:
do you wish to change the image so that BioDrop is in the header of the newly created profile?

**Comment by SaraJaoude** [comment]
> I have implemented all of the changes. regarding Step 19: do you wish to change the image so that BioDrop is in the header of the newly created profile?

Upon reflection, I don't think we need the image for Step 19 as it does not add much value. Would you be able to remove please? Thanks!

**Comment by goobric** [comment]
hi @SaraJaoude, a different perspective: if a person is a new user to BioDrop, and have just signed; I think step19 reassures them that their profile has been set up, and where they can access it from. Having your profile picture for step19 I believe adds value, as a new user would be able see what a type of response they would receive once their profile has been merged and active.
What do you think?
Otherwise yes, I can remove this.

**Comment by goobric** [comment]
hi @SaraJaoude 
Note about the images:
- the images that were provided, with BioDrop, I added a red rectangle to those whose words that needed emphasising to the user.
- as I wasn't able to reference these images via a url-link; I wanted to place them within an assets/images directory; but I couldn't locate one. Therefore I created an 'images' directory within the 'docs' folder, and placed the new images in there, and reference them within the mdx file. Sorry if this was not the correct process :-/
- As requested, I removed the current image that was associated with step19

I have submitted the solution to issue 9697, via a PR (9714)

Thank you so much for allowing me to work on this; thank you for the ease of communication and clarification during this process :-)  please thank Eddie for me as well.

**Comment by eddiejaoude** [comment]
> Having your profile picture for step19 I believe adds value

I can create this screenshot if needed, we can always add it back in later if needed also

> as I wasn't able to reference these images via a url-link; I wanted to place them within an assets/images directory; but I couldn't locate one

when you upload the images to the issue in a comment, GitHub generates a url for you that you can use

**Comment by goobric** [comment]
hi @eddiejaoude 
thanks for the feedback.
- As I created a new branch 'qsjson' from the main branch in my cloned repo; main is still intact with the original image for step19; therefore I can put this back into the qsjson branch
- if I understand this correctly; if I upload the new images here in this conversation; github will generate a url-link for each of those images?
- and therefore I can remove the 'images' directory and the images inside this folder?
- once I have done the above; do I then create a new issue for the PR 9714?

**Comment by goobric** [comment]
![BD_qsjson_step19](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/fd8d729d-0681-4479-a87e-a0fccf822589)

**Comment by Abhishek-90** [comment]
> hi @eddiejaoude thanks for the feedback.
> 
> * As I created a new branch 'qsjson' from the main branch in my cloned repo; main is still intact with the original image for step19; therefore I can put this back into the qsjson branch
> * if I understand this correctly; if I upload the new images here in this conversation; github will generate a url-link for each of those images?
> * and therefore I can remove the 'images' directory and the images inside this folder?
> * once I have done the above; do I then create a new issue for the PR 9714?

No @goobric you don't need to create a new issue/pr when you add the changes. 

What you can do is, post the images that needs to be added here in the conversation, for every image Github will give you an URL, use this URL instead of loading the images from `images` folder. Then remove the images from the folder and commit your changes to the same branch. Your pr #9714 will be updated i.e. the images will be removed and the `src` attribute for images will be replaced with the url generated by Github.

**Comment by goobric** [comment]
hi @Abhishek-90 
- thank you for the information and the instructions
- for clarity; I would like to share my understanding: once I amend my 'qsjson' branch locally and save it; and then push it to my forked repo on Github; this will automatically update the PR #9714  ? is that correct understanding?

**Comment by goobric** [comment]
![279361179-e00b3df3-f55e-46ca-bb02-9e3603947f48_03](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/53747c7b-5948-434a-be92-eb58ab26a74b)
![279361179-e00b3df3-f55e-46ca-bb02-9e3603947f48_04](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/d6e7b2df-e7d8-4877-aea5-9821bc8df904)
![279361213-832445a5-b2ed-48d5-a633-82f6a6ec2b54-05a](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/d1b74078-9905-42e7-9b4b-b8c40ebf8425)
![279684953-ad2524c2-1fe3-469a-b5a2-cf436f1fe5f8](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/8b21ad42-3648-4cf8-a76e-909b0af636f1)
![279361309-354c0448-34e2-457e-b117-10c975ee5f69_15](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/2835ed3c-c305-404d-8294-8bde72a937f2)

**Comment by goobric** [comment]
hi @Abhishek-90 
following on from your advice; my forked BioDrop, branch 'qsjson' is 32 commits ahead of EddieHubCommunity:main
see screen image
how do I proceed now?

**Comment by goobric** [comment]
![BD_qsjson_update](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/ff880cdd-a236-4b44-9f0c-0d9b1fb633ba)

**Comment by Abhishek-90** [comment]
> hi @Abhishek-90
> 
> * thank you for the information and the instructions
> * for clarity; I would like to share my understanding: once I amend my 'qsjson' branch locally and save it; and then push it to my forked repo on Github; this will automatically update the PR [Updated the QuickStart-JSON documentation with new BioDrop images #9714](https://github.com/EddieHubCommunity/BioDrop/pull/9714)  ? is that correct understanding?

Yes @goobric , your understanding is correct. 

Made the amends locally -> commit changes -> push those changes to your forked repo (same branch as the pr is raised from) -> your PR #9714 will be updated.

**Comment by Abhishek-90** [comment]
> hi @Abhishek-90 following on from your advice; my forked BioDrop, branch 'qsjson' is 32 commits ahead of EddieHubCommunity:main see screen image how do I proceed now?

For now don't update the main branch of your forked repo. Update the PR and make sure there are no merge conflicts. Once your PR is approved, your code will be merged with the main branch of parent repo i.e. EddiehubCommunity/BioDrop. Now you can sync your forked repo's main branch. 

Another thing you can do is directly pull the commits added to the parent repo, that is upstream repo. This is will pull all the changes added to Parent repo. You can refer to this video to `pull from upstream`

https://youtu.be/4AbJjvMHTZk?si=_bPPr0SBVa1WCZJ9
https://youtu.be/deEYHVpE1c8?si=SQ9ewgxcoJALHDk7

Incase of any merged conflicts. You can resolve it manually or ask community for help.

**Comment by goobric** [comment]
hi @Abhishek-90 
- yesterday, my branch 'qsjson' from the cloned BioDrop, successfully passed all of the tests during the PR process (see src img)
![BD_qsjson_test_error](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/0e77c28a-7bfc-4290-9b07-92d1d96f29b4)
![BD_qsjson_test_pass](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/085d1b52-662a-44a3-9d05-3c02b16a61fa)

- now, since updating the 'qsjson' branch, saving it locally, pushing it to my forked BioDrop repo; it has now failed Test 5!
- what is Test 5? and why did the updated branch fail this test? (see scr img)

**Comment by Abhishek-90** [comment]
> hi @Abhishek-90
> 
> * yesterday, my branch 'qsjson' from the cloned BioDrop, successfully passed all of the tests during the PR process (see src img)
>   ![BD_qsjson_test_error](https://user-images.githubusercontent.com/23141638/280262719-0e77c28a-7bfc-4290-9b07-92d1d96f29b4.PNG)
>   ![BD_qsjson_test_pass](https://user-images.githubusercontent.com/23141638/280262732-085d1b52-662a-44a3-9d05-3c02b16a61fa.PNG)
> * now, since updating the 'qsjson' branch, saving it locally, pushing it to my forked BioDrop repo; it has now failed Test 5!
> * what is Test 5? and why did the updated branch fail this test? (see scr img)

I am not sure about the reason, maybe @eddiejaoude can guide you better.

**Comment by goobric** [comment]
hi @SaraJaoude and @eddiejaoude 
issue #9697 and PR #9714
please excuse my naivety of the intricacies of github;
context: using the branch 'qsjson' from the cloned BioDrop on my local machine.
Since the initial submitted PR (yesterday) I have updated the 'qsjson' branch by:
- removing the image folder and images within it, and using GH generated url-links to reference the images
- added back the image for step19
- I saved my changes locally, then pushed them to my forked BioDrop repo on GH
- Quest: are the changes made on my branch on my forked repo, reflected on the original PR made?
- why has my PR now failed test5? When the PR initially passed all tests!  What is test5?
Sorry for so many questions :-/
Is there is another maintainer / moderator for BioDrop who maybe able to explain and assist me; so I know how extremely busy you both are.

**Comment by Abhishek-90** [comment]
Hi @goobric, seems like there is an issue with Test 5, #9727 this issue is opened which is the same problem you are facing. 

To answer your questions, yes you new changes are getting reflected in your PR #9714. You check click on the file changes or commits tab, on your PR Page

Refer below image.

<img width="632" alt="Screenshot 2023-11-05 081751" src="https://github.com/EddieHubCommunity/BioDrop/assets/43419831/0929685f-4838-4d3f-a67f-d787f91d9e72">

**Comment by SaraJaoude** [comment]
Thanks @Abhishek-90 

@goobric I can see that the tests are now passing
